app/components/ucsf/RefinementLists/EligibilityRefinements.tsx

```diff
@@ -91,74 +87,3 @@ const ClientEligibilities = ({
     </div>
   );
 };
-
```

Comment:
Page component refactored/consolidated with `ServiceType` Page component and  moved to UcsfDiscoveryForm.tsx